test-clusters/cluster2-lab.yaml

```diff
@@ -2,12 +2,13 @@ name: cluster2-lab
 helm:
   log: 0
   debug: true
+  force: true
```

Comment:
i think this parameter should be per release instead of per cluster. We don't want to force replace all releases

Reply:
### ./impeller --cluster-config-path=test-clusters/cluster2-lab.yaml --kube-config="$(cat ~/.kube/config)" --kube-context colima --dry-run

```
name: cluster2-lab
helm:
  log: 0
  debug: true
  repos:
    - name: stable
      url: https://charts.helm.sh/stable
releases:
  - name: istio-base
    namespace: kube-system
    version: ~x.x.x
    chartPath: "./downloads/istio-1.11.5/manifests/charts/base"
    chartsSource: "https://github.com/istio/istio/releases/download/1.11.5/istio-1.11.5-linux-amd64.tar.gz"
  - name: istio-ingress
    namespace: kube-system
    version: ~x.x.x
    chartPath: "./downloads/istio-1.11.5/manifests/charts/gateways/istio-ingress"
    chartsSource: "https://github.com/istio/istio/releases/download/1.11.5/istio-1.11.5-linux-amd64.tar.gz"
    force: true
```
### with --force

```
2022/02/11 20:42:30 Installing addon: istio-ingress @ ~x.x.x
2022/02/11 20:42:30 Running with Force: istio-ingress
2022/02/11 20:42:30 Charts Source defined for: istio-ingress
2022/02/11 20:42:30 File exist, skipping download: ./downloads/istio-1.11.5-linux-amd64.tar.gz
2022/02/11 20:42:30 Running Dry run: istio-ingress
2022/02/11 20:42:30 RUNNING: helm upgrade --install --force istio-ingress ./downloads/istio-1.11.5/manifests/charts/gateways/istio-ingress --version ~x.x.x --kube-context colima --debug --namespace kube-system --dry-run
history.go:56: [debug] getting history for release istio-ingress
Release "istio-ingress" does not exist. Installing it now.
install.go:178: [debug] Original chart version: "~x.x.x"
install.go:195: [debug] CHART PATH: /Users/Z0035RX/github/kannanthiru/impeller/downloads/istio-1.11.5/manifests/charts/gateways/istio-ingress
```

### without --force

```
2022/02/11 20:42:28 Installing addon: istio-base @ ~x.x.x
2022/02/11 20:42:28 Charts Source defined for: istio-base
2022/02/11 20:42:28 File exist, skipping download: ./downloads/istio-1.11.5-linux-amd64.tar.gz
2022/02/11 20:42:28 Running Dry run: istio-base
2022/02/11 20:42:28 RUNNING: helm upgrade --install istio-base ./downloads/istio-1.11.5/manifests/charts/base --version ~x.x.x --kube-context colima --debug --namespace kube-system --dry-run
history.go:56: [debug] getting history for release istio-base
Release "istio-base" does not exist. Installing it now.
install.go:178: [debug] Original chart version: "~x.x.x"
install.go:195: [debug] CHART PATH: /Users/Z0035RX/github/kannanthiru/impeller/downloads/istio-1.11.5/manifests/charts/base

install.go:210: [debug] WARNING: This chart or one of its subcharts contains CRDs. Rendering may fail or contain inaccuracies.
NAME: istio-base
LAST DEPLOYED: Fri Feb 11 20:42:29 2022
NAMESPACE: kube-system
STATUS: pending-install
REVISION: 1
TEST SUITE: None
USER-SUPPLIED VALUES:
```